### ISSUE TITLE
Pass along posargs in all invocations of pytest in tox.ini

### ISSUE THREAD

**Issue by MridulS** [scipp/copier_template#164]
Currently it is only pass this in the main test enviornment but we should just pass `{posargs}` along in all of them.

**Comment by YooSunYoung** [comment]
You mean for nightly or unpinned...?
I think I also had some inconveniences as well but I don't remember what exactly...
Can you show us some examples... ?

**Comment by MridulS** [comment]
> You mean for nightly or unpinned...?

Yup exactly, was trying to run and debug the nightly failures locally. Done in https://github.com/scipp/copier_template/pull/165

**Comment by YooSunYoung** [comment]
But if you failed the tox command, shouldn't it be just part of the command as well...?

Regardlessly I think it'll be useful to have the posargs thou.

**Comment by nvaytet** [comment]
Has this been fixed in #165?
